modules/odc-access-web-terminal.adoc

```diff
@@ -6,7 +6,7 @@
 [id="odc-access-web-terminal_{context}"]
 = Accessing the web terminal
 
-After the {web-terminal-op} is installed, you can access the web terminal.
+After the {web-terminal-op} is installed, you can access the web terminal. After the web terminal is initialized, you can use the preinstalled CLI tools like `oc`, `kubectl`, `odo`, `kn`, `tkn`, `helm`, `kubens`, `subctl`, and `kubectx` in the web terminal.
```

Comment:
*PEER REVIEW* Moved this sentence unchanged from [line 30](https://github.com/openshift/openshift-docs/pull/67647/files#diff-3280d02527d34051c6d259501fa40e60cdf0fc555e2183f992e78da26dcf2fffL30).